src/goalServices/goals.js

```diff
@@ -2328,6 +2330,7 @@ export function verifyAllowedGoalStatusTransition(oldStatus, newStatus, previous
  */
 export async function updateGoalStatusById(
   goalIds,
+  userId,
```

Comment:
`GoalStatusChanges` needs to know who performed the action, so I thought this was as a good a place as any to receive this value.